wikidatarefisland/external_identifiers/generate_whitelisted_ext_ids.py

```diff
@@ -0,0 +1,93 @@
+import concurrent.futures
+
+import requests
+
+
+class GenerateWhitelistedExtIds():
+
+    def __init__(self, wdqs_reader, storage, config, external_identifier_formatter,
+                 no_checks=10, no_processes=10):
+        """
+        Implementation of Side-step 1
+
+        :type wdqs_reader: wikidatarefisland.data_access.WdqsReader
```

Comment:
Cool; I didn't know about this type of doc blocks. Looks awesome; I'll try to add this to my PRs

Reply:
IntelliJ suggested :D